mbed_lib.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "core",
+    "config": {
+        "stdio-convert-newlines": {
+            "help": "Enable conversion to standard newlines on stdin/stdout"
```

Comment:
This is technically OK, but I'd rather add a `"value": 0` here to make it clear that the conversion is disabled by default,  and change the condition in `retarget.cpp` to `#if MBED_CONF_CORE_STDIO_CONVERT_NEWLINES`. Not a must, it just feels a bit more clear.